### ISSUE TITLE
Current state of this project

### ISSUE THREAD

**Issue by jordiae** [huggingface/accelerate#211]
Hi, I'm curious about your current intentions regarding this project, since it seems the repo hasn't been especially active since the Deepspeed addition. Are you planning to keep adding new features, or just maintaining the current ones? Thanks!

**Comment by sgugger** [comment]
This project is still active, we just haven't had much time to dive into it with the release of the [Hugging Face course](https://huggingface.co/course/chapter1/1). There was the addition of the dispatch dataloader since the deepspeed integration and I actually have some bugs to fix this morning.

Plans forward are to polish the deepspeed integration, add an easier API to do checkpointing or gradient accumulation, add support for more backends :-)

**Comment by sgugger** [comment]
Yes, it's about the current features first. We will look at the model parallelism ones later down the road :-)
